src/main/python/analyzers/project_analyzer.py

```diff
@@ -62,16 +87,60 @@ def __performRegressionAnalysis(self, dfWithFeatures, featureColumnIndexes):
         @rtype: dictionary
         """
         # Perform regression analysis
-        # return resultsDictOrDF
+        X = dfWithFeatures[featureColumnIndexes]
+        y = dfWithFeatures['PlusMinusPoints']
+        X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.33, random_state=42)
+
+        # Generating models
+        lr = LinearRegression()
+        lr.fit(X_train, y_train)
+        prediction = lr.predict(X_test)
+        resultsDictOrDF = pd.concat([y_test, pd.Series(prediction, index=y_test.index, name='y_test')], axis=1)
+        resultsDictOrDF.columns = ['y_pred', 'y_test']
+        return resultsDictOrDF
+
+    def __performMLPAnalysis(self, dfWithFeatures, featureColumnIndexes):
+        """
+        Performs whatever other analysis after the initial analysis
+
+        @param dfWithFeatures: The data frame that will have the features from an analysis
+        @param featureColumnIndexes: A list of the indexes that contain the feature columns
+        @return: A dataframe of measurements of how well the other model worked for the given dataframe
+        @rtype: Dataframe
+        """
+        # Perform regression analysis
+        X = dfWithFeatures[featureColumnIndexes]
+        y = dfWithFeatures['PlusMinusPoints']
+        X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.33, random_state=42)
+
+        # Generating models
+        mlpc = MLPRegressor(hidden_layer_sizes=(100, 200, 100), activation='relu',
+                            max_iter=10000, solver='adam', alpha=0.005, learning_rate_init=0.001,
+                            shuffle=False)
+        mlpc.fit(X_train, y_train)
+        prediction = mlpc.predict(X_test)
+        resultsDictOrDF = pd.concat([y_test, pd.Series(prediction, index=y_test.index, name='y_test')], axis=1)
+        resultsDictOrDF.columns = ['y_pred', 'y_test']
+        return resultsDictOrDF
+
+    def __performRandomForestAnalysis(self, dfWithFeatures, featureColumnIndexes):
+        """
+        Performs whatever other analysis after the initial analysis
```

Comment:
Just make sure to go through the comments here too. Looks like a copy paste error.

Reply:
done